### PR TITLE
[google_maps_flutter] Clean up unnecessary code

### PR DESCRIPTION
**Pull request by seungsoo47**
* Clean up unnecessary code (like warnings suggested by VSCode, or print statements for debugging)
* Update README.md & CHANGELOG.md files
* Fix bugs

### CODE REVIEW

**Comment by bbrto21** [comment]
Could you rebase master? I want to see whether #199 works properly or not

File: packages/google_maps_flutter/example/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 name: google_maps_flutter_tizen_example
 description: Demonstrates how to use the google_maps_flutter_tizen plugin.
-publish_to: none
```
Comment (bbrto21) [inline packages/google_maps_flutter/example/pubspec.yaml:3]: Why did you remove this?
https://dart.dev/tools/pub/pubspec#publish_to

**Review by bbrto21 — CHANGES_REQUESTED** [review]
Please, rebase master first and resolve my comments

File: packages/google_maps_flutter/README.md [hunk 2]
```diff
@@ -28,6 +28,6 @@ In addition, you need a Maps JavaScript API Key to use this plugin. Please refer
 To use 'newCameraPosition' API, please add 'v=beta' option in map.html as shown below:
 ```
 <script async
-    src="https://maps.googleapis.com/maps/api/js?v=beta&key=YOUR_API_KEY&callback=initMap">
+    src="https://maps.googleapis.com/maps/api/js?v=beta&key=YOUR_API_KEY">
```
[the remark this answers is not in the document]
Comment (seungsoo47) [reply packages/google_maps_flutter/README.md:31]: 'newCameraPosition' API is implemented based on 'moveCamera' javascript API of Google Map.
https://developers.google.com/maps/documentation/javascript/reference/map#Map.moveCamera

File: packages/google_maps_flutter/example/pubspec.yaml (hunk 1 above)
Comment (seungsoo47) [reply packages/google_maps_flutter/example/pubspec.yaml]: Oh. It's my mistake.

**Review by bwikbs — APPROVED** [review]
A version that solves the problem when there are two maps at same view(#213), so please raise the webvew_flutter_tizen version sufficiently when releasing. :smile:

**Comment by seungsoo47** [comment]
@bbrto21 I have updated for your comments. Please review again.

File: packages/google_maps_flutter/pubspec.yaml [hunk 3]
```diff
@@ -15,9 +15,10 @@ dependencies:
     sdk: flutter
   meta: ^1.3.0
   google_maps_flutter_platform_interface: ^2.0.1
+  flutter_lints: ^1.0.4
   stream_transform: ^2.0.0
   webview_flutter: 2.0.4
-  webview_flutter_tizen: ^0.3.0
+  webview_flutter_tizen: ^0.3.2
```
Comment (bbrto21) [inline packages/google_maps_flutter/pubspec.yaml:21]: 0.3.3 was recently released.

**Review by bbrto21 — APPROVED** [review]
LGTM
